synapse/handlers/room_member.py

```diff
@@ -1405,7 +1405,7 @@ async def do_3pid_invite(
             # We don't check the invite against the spamchecker(s) here (through
             # user_may_invite) because we'll do it further down the line anyway (in
             # update_membership_locked).
-            _, stream_id = await self.update_membership(
+            event_id, stream_id = await self.update_membership(
```

Comment:
Ahh: the mistake was that we didn't have an `event` to return in this case? Sorry for missing this.

Reply:
Nw at all, thanks for the review!